### ISSUE TITLE
docs: skip links to content in AIO for accessibility

### ISSUE THREAD

**Issue by twerske** [angular/angular#42936]
### Description

angular.io should implement skip links to allow developers using assistive technology to skip navigation-related parts of the page which are presented before the main content. 

This is a [web best practice recommendation](https://www.w3.org/TR/WCAG20-TECHS/G1.html) and will improve the core accessibility of our documentation.

### What is the affected URL?

angular.io

### Please provide the steps to reproduce the issue

1. Turn on a screen reader (I'm using VoiceOver on a mac)
2. Navigate through AIO to any docs page
3. Attempt to skip past menus and overview to the page content

### Please provide the expected behavior vs the actual behavior you encountered

<a href="#content" ref="skipLink">Skip to main content</a>

**Comment by jelbourn** [comment]
I'm recategorizing this from a feature to a bug because missing a skip link is an accessibility problem.
